pyspecProcScripts/correlation_alignment.py

```diff
@@ -81,20 +82,19 @@ def correl_align(
 
     logging.debug(psd.strm("Applying the correlation routine"))
     if avg_dim:
-        phcycdims = [j for j in s_orig.dimlabels if j.startswith("ph")]
-        indirect = set(s_orig.dimlabels) - set(phcycdims) - set([direct])
-        indirect = [j for j in s_orig.dimlabels if j in indirect]
-        avg_dim_len = len(s_orig.getaxis(avg_dim))
-        s_orig.smoosh(indirect)
-    for j in signal_pathway.keys():
-        assert not s_orig.get_ft_prop(j), (
-            str(j) + " must not be in the coherence domain"
+        phcycdims = [j for j in s_jk.dimlabels if j.startswith("ph")]
+        indirect = set(s_jk.dimlabels) - set(phcycdims) - set([direct])
+        indirect = [j for j in s_jk.dimlabels if j in indirect]
+        s_jk.smoosh(indirect)
+    for phnames in signal_pathway.keys():
+        assert not s_jk.get_ft_prop(phnames), (
+            str(phnames) + " must not be in the coherence domain"
         )
     signal_keys = list(signal_pathway)
     signal_values = list(signal_pathway.values())
-    ph_len = {j: psd.ndshape(s_orig)[j] for j in signal_pathway.keys()}
-    N = psd.ndshape(s_orig)[indirect_dim]
-    sig_energy = (abs(s_orig) ** 2).data.sum().item() / N
+    ph_len_dict = {j: psd.ndshape(s_jk)[j] for j in signal_pathway.keys()}
```

Comment:
specify this is a dictionary containing the lengths of ph cycling dims

Reply:
again, I do not see the benefit to renaming  at this point --> your naming of `s_jk` **absolutely** makes sense later, when you have what I have been calling the "left" and the "right".  But I see this as just complicating the diff for these earlier/setup functions.

I know that I do discourage copies where not needed, but I also really see the benefit here of an `s_orig`, which is *not to be modified* since that's the way the code is written.